projects/js-packages/components/components/threats-data-views/index.tsx

```diff
@@ -68,7 +73,9 @@ export default function ThreatsDataViews( {
 	onFixThreats,
 	onIgnoreThreats,
 	onUnignoreThreats,
+	header,
 }: {
+	historic?: boolean;
```

Comment:
What do you think about using a const type for this prop, something like `type: 'current' | 'historic'`?

Motivation being to keep things flexible for the future.